build/template-ios-appcenter-tests.yaml

```diff
@@ -3,17 +3,19 @@ parameters:
 
 steps:
 
-- task: DownloadBuildArtifacts@0
+- task: DownloadPipelineArtifact@2
   displayName: 'Download Xamarin.iOS'
   inputs:
-    artifactName: Xamarin.iOS
-    itemPattern: '**/*'
+    artifact: Xamarin.iOS
+    patterns: '**/*'
+    path: $(Build.artifactstagingdirectory)/drop
```

Comment:
Should this be:
```suggestion
    path: $(Build.artifactstagingdirectory)/Xamarin.iOS
```

Reply:
It probably should be, yes. There is no iOS pipeline that I can test this with, that's probably why I did not notice. Will fix, thanks!

Reply:
Sounds good. Thanks. Yea, we disabled it for a while since doesn't mesh with some net6 mobile targets. We can test once we reenable it.